app/main.py

```
@@ -324,7 +324,48 @@ async def download_event_ics(event_id: str, client: httpx.AsyncClient = Depends(
+# Fetch one copyright info for images from the CMS
+
+
+async def fetch_one_copyright(image_id: str, client: httpx.AsyncClient):
+    params = {
+        "filters[media][documentId][$eq]": image_id,
+        "pagination[page]": 1,
+        "pagination[pageSize]": 1,
+        "populate[media]": "true",
+    }
+
+    copyright_json, _ = await _cms_get(f"/copyrights", params, client)
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)